### ISSUE TITLE
Content Category taxonomy

### ISSUE THREAD

**Issue by RobJohnston** [wet-boew/wet-boew-drupal#516]
Ran into a problem while importing data with the migrate module and noticed a bit of a tangle with the built-in "categories" taxonomy.  The content type "Basic Page" has a "Categories" field which does not show when adding/editing a page but I can make it show if I simply go to /admin/structure/types/manage/wetkit-page/fields/field_featured_categories and click the "Save settings" button.  Haven't dug in very far but my first guess is that there is a machine name conflict somewhere during the install between "panalopy_categories" and "wetkit_categories"?

**Comment by sylus** [comment]
Hey @RobJohnston you are correct. I will push a fix later in the day :)

Thanks for the catch!

**Comment by RobJohnston** [comment]
Might be a similar thing happening for "Feature content" (field_featured_status), as I don't see that on the "Basic Page" but it is defined as a field.

**Comment by RobJohnston** [comment]
I'm going to add one more thing as it is a problem that will show once you get the first item done, and since you're going to be in there anyways...  

When a page is tagged with a taxonomy the term(s) show at the bottom of the page, but it looks like a field label is missing as one just sees a colon followed by the term.  That is, I would have expected to see something like "Categories:  News Releases" instead of ":  News Releases".  

I've tracked this down to $vars['label'] being empty in the "adaptivetheme_field__taxonomy_term_reference" function in /sites/all/themes/adaptivetheme/at_core/ince/theme.inc.  But what to do... hmmm.

**Comment by sylus** [comment]
Hmm @jmburnz thoughts?

**Comment by sylus** [comment]
@RobJohnston could this be related to fences being enabled?

**Comment by RobJohnston** [comment]
@sylus, I'm going to say no... I disabled the fences module (had to disable WetKit WET-BOEW module as well), even threw in a cache clear, and there was no change to the html produced:
<code>
&lt;h2 class="field-label"&gt;:&nbsp;&lt;/h2&gt;
</code>
Any other test I can do?

**Comment by sylus** [comment]
Hmm have it duplicated and filed an issue upstream. http://drupal.org/node/1848044

**Comment by RobJohnston** [comment]
I'm looking at an older version of drupal-wet but I found what is hiding the 'Categories' label from showing up.  It was on the /profiles/wetkit/modules/custom/wetkit_pages/wetkit_pages.panelizer.inc file, specifically the line number 399 in this version, under 
<code>$pane->subtype = 'node:field_featured_categories';</code>
There's a configuration array below that that has 'override_title => 1 and nothing in override_title_text.  Change the one to a zero and you'll see the "Categories:" label.  

Now working on getting the taxonomy terms to show up in French.

**Comment by RobJohnston** [comment]
... and for showing the terms in French, that same configuration array as mentioned above should be changed from 
<code>'formatter' => 'taxonomy_term_reference_link',<code> to
<code>'formatter' => 'taxonomy_term_reference_link_localized',<code>.
Booyah!

**Comment by RobJohnston** [comment]
Can somebody with experience in panelizer look at my patch suggestion.  It works, but I don't have reason to believe that it's correct.  I noticed that if I suffix taxonomy_term_reference_link with either _localized or _localize it still works, so I went searching for the most correct way of doing it and came up with nothing.  Then I noticed that if I just suffix taxonomy_term_reference_link with any character (underscore, the letter x, a space) it will still work.
